torch/_inductor/codegen/triton_combo_kernel.py

```diff
@@ -0,0 +1,858 @@
+# mypy: allow-untyped-defs
```

Comment:
Can we remove this in a follow up ?

Reply:
Sure I will remove it in the 2nd pull request shortly.